test/ruby/test_zjit.rb

```diff
@@ -479,6 +479,17 @@ def fib(n)
     }, call_threshold: 5, num_profiles: 3
   end
 
+  # tool/ruby_vm/views/*.erb relies on the zjit instructions a) being contiguous and
+  # b) being reliably ordered after all the other instructions.
+  def test_instruction_order
+    zjit, others = RubyVM::INSTRUCTION_NAMES.map.with_index.partition { |name, _| name.start_with?('zjit_') }
+    zjit_indexes = zjit.map(&:last)
+    other_indexes = others.map(&:last)
+    zjit_indexes.product(other_indexes).each do |zjit_index, other_index|
+      assert zjit_index > other_index, "ZJIT instruction at #{zjit_index} must be defined after other instruction at #{other_index}"
```

Comment:
If you feel inclined, we can use the name in the error message instead of tossing it away

Reply:
:+1: https://github.com/ruby/ruby/pull/13199/commits/26b9a45917c837a9c6f42840aaee7179f42d1ca8